target/linux/qualcommax/files/arch/arm64/boot/dts/qcom/ipq8074-x80-5g.dts

```diff
@@ -0,0 +1,555 @@
+		 serial0 = &blsp1_uart5;
+ 		 ethernet0 = &dp5; // Aliases to capture wan mac from bootloader
+		 ethernet1 = &dp6_syn; // Aliases to capture lan mac  from bootloader (currently unused as mac patched from factory data partition)
+	};
+
+	chosen {
+		stdout-path = "serial0:115200n8";
+		bootargs-append = " root=/dev/ubiblock0_1";
+	};
+
+	modem {
+		compatible = "gpio-modem";
+		pinctrl-0 = <&modem_pins>;
+		pinctrl-names = "default";
+
+		w_disable {
+			label = "w_disable";
+			gpios = <&tlmm 35 GPIO_ACTIVE_HIGH>; // wwan disable output
+		};
+
+		modem-reset {
+			label = "modem_reset";
+			gpios = <&tlmm 29 GPIO_ACTIVE_HIGH>; // reset modem output
+		};
+	};
+
+		antennas {
+		compatible = "gpio-antenna";
+		pinctrl-0 = <&antenna_pins>;
+		pinctrl-names = "default";
+
+		ext_antenna_en {
+			label = "ext_antenna_en";
+			gpios = <&tlmm 56 GPIO_ACTIVE_HIGH>; // select external sma antennas for WWan output
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		pinctrl-0 = <&button_pins>;
+		pinctrl-names = "default";
+
+		button@1 {
+			label = "reset";
+			gpios = <&tlmm 27 GPIO_ACTIVE_LOW>; //reset button output
+			linux,code = <KEY_RESTART>; // Aliases for button to stitch to linux reset functioon
+		};
+	};
+
+	led_spi {
+		compatible = "spi-gpio";
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		sck-gpios = <&tlmm 18 GPIO_ACTIVE_HIGH>; //clock signal for spi connected led's
+		mosi-gpios = <&tlmm 19 GPIO_ACTIVE_HIGH>; //serial data for spi connected led's
+
+		led_gpio: led_gpio@0 {
+			compatible = "fairchild,74hc595";
+			reg = <0>;
+			gpio-controller;
+			#gpio-cells = <2>;
+			registers-number = <2>;
+			enable-gpios = <&tlmm 20 GPIO_ACTIVE_HIGH>; // powersupply for spi mux chip
+			spi-max-frequency = <1000000>;
+		};
+	};
+
+	phy-leds {
+		compatible = "gpio-leds";
+
+		led_wifi_son_green: led0 {
+			gpios = <&led_gpio 0 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_wifi_son_orange: led1 {
+			gpios = <&led_gpio 1 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_ORANGE>;
+		};
+
+		led_wifi_son_blue: led2 {
+			gpios = <&led_gpio 2 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_BLUE>;
+		};
+
+		led_2g: led3 {
+			gpios = <&led_gpio 3 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_GREEN>;
+			linux,default-trigger = "phy1radio";
+		};
+
+		led_5gl: led4 {
+			gpios = <&led_gpio 4 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_5gh: led5 {
+			gpios = <&led_gpio 5 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_lan: led6 {
+			gpios = <&led_gpio 6 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_wan: led7 {
+			gpios = <&led_gpio 7 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_GREEN>;
+			linux,default-trigger = "none";
+		};
+
+		led_10g_link: led8 {
+			gpios = <&led_gpio 8 GPIO_ACTIVE_LOW>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_sfp: led9 {
+			gpios = <&led_gpio 9 GPIO_ACTIVE_HIGH>;
+			color = <LED_COLOR_ID_GREEN>;
+		};
+
+		led_bt: led10 {
```

Comment:
Please drop unused DT labels (`led_*: `).

Reply:
ok I'm just following the config from Qualcom the SPI LED's are not present on this device but are present on the OEM config, it would mean it would be compatible with other boards running the same configuration.